authentik/policies/models.py

```diff
@@ -142,6 +149,9 @@ def __str__(self) -> str:
             return f"Binding - #{self.order} to {suffix}"
         return ""
 
+    objects = models.Manager()
+    in_use = BoundPolicyQuerySet.as_manager()
```

Comment:
```suggestion
    objects = BoundPolicyQuerySet.as_manager()
```

Not fully sure about how it's done elsewhere, but I think this is the most common pattern I've seen in the codebase. Just a nitpick though, if you feel it makes sense to have it as a separate manager, fine by me.

Reply:
I prefer keeping this a separate manager for now, given it's our first instance of needing to check actively used policies.